src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -101,9 +101,6 @@ public class HttpUtil {
 
   private static boolean socksProxyDisabled = false;
 
-  /** customized proxy properties */
-  static boolean httpUseProxy = false;
```

Comment:
why remove this package level variable? Do we know no one using even with reflection?

Reply:
Right, this is completely unused. It's just an extra variable.